lang/src/gen/build.rs

```diff
@@ -70,7 +70,7 @@ fn codegen_for_contract_env(tokens: &mut TokenStream2, contract: &hir::Contract)
             Moment,
         };
 
-        #[allow(snake_case)] type env = types::ContractEnv;
+        #[allow(snake_case)] type env = types::_ContractEnv;
```

Comment:
Why?

Reply:
This was a temporary change to fix an issue with conflicting names, shouldn't have been pushed